contracts/other/lido/StEthConverter.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import "@yield-protocol/utils-v2/contracts/token/IERC20.sol";
+import "@yield-protocol/utils-v2/contracts/token/TransferHelper.sol";
+import "../../oracles/lido/IWstETH.sol";
+
+/// @dev A contract to handle wrapping & unwrapping of stETH
+contract StEthConverter {
+    using TransferHelper for IERC20;
+    using TransferHelper for IWstETH;
+
+    IWstETH public immutable wstETH;
+    IERC20 public immutable stETH;
+
+    constructor(IWstETH wstETH_, IERC20 stETH_) {
+        wstETH = wstETH_;
+        stETH = stETH_;
+        stETH_.approve(address(wstETH_), type(uint256).max);
+    }
+
+    /// @dev Wrap stEth held by this contract and forward it to the "to" address
+    function wrap(address to) external returns (uint256 wstEthAmount) {
+        uint256 stEthAmount = stETH.balanceOf(address(this));
+        require(stEthAmount > 0, "No stETH to wrap");
```

Comment:
this check is redundant shall we remove it?
![Screen Shot 2022-04-20 at 8 27 16 AM](https://user-images.githubusercontent.com/71567643/164266891-5b626f72-be3f-45a2-846e-da26f860b56f.png)
https://etherscan.io/address/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0#code